File: app/src/main/java/me/kirimin/mitsumine/db/AccountDAO.java
package me.kirimin.mitsumine.db;

import com.activeandroid.query.Select;

import me.kirimin.mitsumine.model.Account;

public class AccountDAO {

    public static void save(Account account) {
        account.save();
    }

    public static void delete() {
        Account account = new Select().from(Account.class).executeSingle();
        if (account != null) {
            account.delete();
        }
    }

    public static Account get() {
        return new Select().from(Account.class).executeSingle();
    }
}

File: app/src/main/java/me/kirimin/mitsumine/db/AccountDAO.kt
package me.kirimin.mitsumine.db

import com.activeandroid.query.Select

import me.kirimin.mitsumine.model.Account

public class AccountDAO {
    companion object {

        public fun save(account: Account) {
            account.save()
        }

        public fun delete() {
            val account = Select().from(javaClass<Account>()).executeSingle<Account>()
            if (account != null) {
                account.delete()
            }
        }


        public fun get(): Account? {
            return Select().from(javaClass<Account>()).executeSingle<Account>()
        }
    }
}
